thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -1067,14 +1067,14 @@ private boolean filterConfigsBy(Map<String, Object> detectionConfig, String data
     List<Map<String, Object>> yamls;
     if (dataset == null && metric == null) {
       yamls = this.detectionConfigDAO
-          .findAll()
+          .list(100, 0)
```

Comment:
can you update the java-doc of this helper method as well!

Reply:
Thanks. updated